### ISSUE TITLE
Error occurs if package is missing

### ISSUE THREAD

**Issue by dennismende** [stefanpenner/hash-for-dep#40]
```
Cannot read property 'name' of null        
TypeError: Cannot read property 'name' of null        
at again (/cache/node_modules/hash-for-dep/lib/deps-for.js:19:25)        
at /cache/node_modules/hash-for-dep/lib/deps-for.js:27:7        
at Array.forEach (native)        
 at again (/cache/node_modules/hash-for-dep/lib/deps-for.js:26:55)        
at /cache/node_modules/hash-for-dep/lib/deps-for.js:27:7        
at Array.forEach (native)        
at again (/cache/node_modules/hash-for-dep/lib/deps-for.js:26:55)        
at /cache/node_modules/hash-for-dep/lib/deps-for.js:27:7        
at Array.forEach (native)        
at again (/cache/node_modules/hash-for-dep/lib/deps-for.js:26:55)
```

**Comment by rwjblue** [comment]
Thanks for reporting, this is essentially the same issue as https://github.com/ember-cli/ember-cli/issues/7390 (as you know).

**Comment by rwjblue** [comment]
Fixed by #41
